SystemManager/Inc/SystemManager.hpp

```diff
@@ -16,27 +16,35 @@
 #define SBUS_MAX 100
 
 class SystemManager {
-public:
-    /* Constructors and Destructors */
-    SystemManager();
-    ~SystemManager();
-
-    /* Other managers*/
-    TelemetryManager *telemetryManager;
-
-    /* Class Functions */
-    void flyManually();
-
-private:
-    SBUSReceiver* rcController_;
-    RCControl rcInputs_;
-    PWMChannel throttleMotorChannel_;
-    PWMChannel yawMotorChannel_;
-    PWMChannel rollMotorChannel_;
-    PWMChannel invertedRollMotorChannel_;
-    PWMChannel pitchMotorChannel_;
-    IndependentWatchdog watchdog_;
+    public:
+        /* Constructors and Destructors */
+        SystemManager();
+        ~SystemManager();
```

Comment:
empty destructor definition in source file. consider removing the declaration and definition, making it implicit, or flagging with "= delete"